test/command/config.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { writeFileSync } from 'fs'
+import cli from 'furious-commander'
+import { join } from 'path'
+import { optionParameters, rootCommandClasses } from '../../src/config'
+
+describe('Test configuration loading', () => {
+  let consoleMessages: string[] = []
+
+  const configFolderPath = join(__dirname, '..', 'testconfig')
+  const configFileName = 'config.config.json'
+  const configFilePath = join(configFolderPath, configFileName)
+
+  beforeAll(() => {
+    global.console.log = jest.fn(message => {
+      consoleMessages.push(message)
+    })
+    jest.spyOn(global.console, 'warn')
+    //set config environment variable
+    process.env.SWARM_CLI_CONFIG_FOLDER = configFolderPath
+    process.env.SWARM_CLI_CONFIG_FILE = configFileName
```

Comment:
does it not conflict with the other test runs?

Reply:
Test suites are supposed to have isolated envs